src/test/scripts/README.md

```diff
@@ -1,33 +1,59 @@
-
 # check-merge-to-enterprise Job
 
-When you open a PR on community, if it creates a conflict with enterprise-master, the check-merge-to-enterprise will fail. Say your branch name is `X`, we will refer to `X` on community as `community/X` and on enterprise as `enterprise/X`. If the job already passes, you are done, nothing further required! Otherwise follow the below steps:
+When you open a PR on community, if it creates a conflict with
+enterprise-master, the check-merge-to-enterprise will fail. Say your branch name
+is `$PR_BRANCH`, we will refer to `$PR_BRANCH` on community as
+`community/$PR_BRANCH` and on enterprise as `enterprise/$PR_BRANCH`. If the
+job already passes, you are done, nothing further required! Otherwise follow the
+below steps. First make sure these two things are the case:
+
+1. Get approval from your reviewer for `community/$PR_BRANCH`. Only follow the
+   next steps after you are about to merge the branch to community master.
+2. Make sure your commits are in a nice state, since you will not be able to do
```

Comment:
Maybe here mention that if `community/$PR_BRANCH` is squash merged then there will be duplicate commits on the enterprise side.